web/cobrands/tfl/base.scss

```diff
@@ -144,7 +144,8 @@ input.form-error, textarea.form-error {
 
 a#geolocate_link {
     font-family: $heading-font;
-    font-size: 1.125em;
+    font-size: 1em;
+    border-radius: 22.5px;
```

Comment:
This is a suspiciously specific measurement! Are you doing it because you want a circular side on a 45px high button? If so, I’d use some really big value like `border-radius: 10em` – the browser will scale it down to fit at most half the height (or width) of the button.